great_expectations/render/renderer/site_index_page_renderer.py

```diff
@@ -17,13 +17,13 @@ def _generate_data_asset_table_section(cls, data_asset_name, link_lists_dict):
         if profiling_links: column_count += 1
         validation_links = link_lists_dict["validation_links"]
         if validation_links: column_count += 1
-        expectation_suite_links = link_lists_dict["expectation_suite_links"]
-        if expectation_suite_links: column_count += 1
+        expectations_links = link_lists_dict["expectations_links"]
```

Comment:
For state for me since I'm not much in this one: is this a substantive change or was it actually a link to expectations (not full suites?)

Reply:
@eugmandel made this change. Help us out?